Code/GraphMol/SmilesParse/CXSmilesOps.cpp

```diff
@@ -1113,6 +1114,71 @@ bool parse_wedged_bonds(Iterator &first, Iterator last, RDKit::RWMol &mol,
   return true;
 }
 
+template <typename Iterator>
+bool parse_doublebond_stereo(Iterator &first, Iterator last, RDKit::RWMol &mol,
+                             unsigned int startAtomIdx,
+                             unsigned int startBondIdx,
+                             Bond::BondStereo stereo) {
+  // these look like: C1CCCC/C=C/CCC1 |ctu:5|
+  // also c and t for cis or trans
+  //
+  while (first < last && *first != ':') {
+    ++first;
+  }
+  if (first >= last || *first != ':') {
+    return false;
+  }
+  ++first;
+
+  while (first < last && *first >= '0' && *first <= '9') {
+    unsigned int bondIdx;
+    if (!read_int(first, last, bondIdx)) {
+      return false;
+    }
+    if (!VALID_BNDIDX(bondIdx)) {
+      BOOST_LOG(rdWarningLog)
+          << "bad bond index in c/t/ctu block " << bondIdx << std::endl;
+      return false;
+    }
+    auto bond = mol.getBondWithIdx(bondIdx + startBondIdx);
+
+    // the cis/trans/unknown marker is relative to the lowest numbered atom
+    // connected to the lowest numbered double bond atom and the
+    // highest-numbered atom connected to the highest-numbered double bond atom
+    // find those
+    auto begAtom = bond->getBeginAtom();
+    auto endAtom = bond->getEndAtom();
+    if (begAtom->getIdx() > endAtom->getIdx()) {
+      std::swap(begAtom, endAtom);
+    }
+    if (begAtom->getDegree() > 1 && endAtom->getDegree() > 1) {
+      unsigned int begControl = mol.getNumAtoms();
+      for (auto nbr : mol.atomNeighbors(begAtom)) {
+        if (nbr == endAtom) {
+          continue;
+        }
+        begControl = std::min(nbr->getIdx(), begControl);
+      }
+      unsigned int endControl = mol.getNumAtoms();
+      for (auto nbr : mol.atomNeighbors(endAtom)) {
+        if (nbr == begAtom) {
+          continue;
+        }
+        endControl = std::min(nbr->getIdx(), endControl);
```

Comment:
The comment implies that this should be `std::max`. If `std::min` is correct, I think that the swap of begin/end isn't actually doing anything.

Reply:
yeah, there's some copy-pasta here.
Thanks for catching that